go/internal/flag/flag_test.go

```diff
@@ -96,21 +96,21 @@ func TestIsFlagProvided(t *testing.T) {
 	pflag.CommandLine = pflag.NewFlagSet("testFlagSet", pflag.ExitOnError)
 
 	isProvided := IsFlagProvided("testFlag")
-	assert.False(t, isProvided)
+	assert.False(t, isProvided, "expected IsFlagProvided to return false as provided flag doesn't exist, got true")
```

Comment:
i'm being _extremely_ nitpicky here (sorry!), but this is a bit too verbose and also redundant. the output on failure reads like this:

```
--- FAIL: TestIsFlagProvided (0.00s)
        	Error:      	Should be true
        	Test:       	TestIsFlagProvided
        	Messages:   	expected IsFlagProvided to return false as provided flag doesn't exist, got true
FAIL
```

i think instead something like this would end up reading better:

```go
flagName := "testFlag"
assert.Falsef(t, IsFlagProvided(flagName), "flag %q should not exist", flagName)
```

(and same comment for the other instances below)

Reply:
Looking at the failing test output, it surely looks verbose. Simplified the error messages.